src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheck.java

```diff
@@ -27,16 +27,33 @@
  * <p>
  * Checks if array initialization contains optional trailing comma.
  * </p>
+ *
  * <p>
  * Rationale: Putting this comma in make is easier to change the
  * order of the elements or add new elements on the end.
  * </p>
+ *
+ * <pre>
+ * Properties:
+ * </pre>
+ * <table summary="Properties" border="1">
```

Comment:
Let's avoid tables in javadoc for now. When we were testing these out, some IDEs didn't make the tables that easy to read.
Use `AnnotationLocationCheck` as an example.